pkg/memio/ioport_linux_amd64.s

```diff
@@ -0,0 +1,60 @@
+// Copyright 2012-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+	// From go runtime compiler source:
+        //TODO: INB DX, AL                      // ec
+        //TODO: INW DX, AX                      // 66ed
+        //TODO: INL DX, AX                      // ed
+
+TEXT ·archInb(SB),$0-8
+        MOVL    port+0(FP), DX
+	BYTE $0xec
+	//INB	DX, AL
+	MOVL    AX, ret+4(FP)
+        RET
+	
+TEXT ·archInw(SB),$0-8
+        MOVL    port+0(FP), DX
+	BYTE $0x66
+	BYTE $0xec
+	//INW	DX, AL
+	MOVL    AX, ret+4(FP)
+        RET
+
+
+TEXT ·archInl(SB),$0-8
+        MOVL    port+0(FP), DX
+	BYTE $0xed
+	//INL	DX, AL
+	MOVL    AX, ret+4(FP)
+        RET
+
+	// From go runtime compiler source:
+        //TODO: OUTB AL, DX                     // ee
+        //TODO: OUTW AX, DX                     // 66ef
```

Comment:
These opcodes don't make sense. Why would you need 2 bytes to encode outw, but only 1 for outb/outl? It's not your fault.

Reply:
comment added.